www/www-deploy.yml

```diff
@@ -137,7 +137,7 @@
     # support
     - match: "~/site/support/?$"
       dest: /docs
-    - match: "~/site/support/ome-artwork/?$"
+    - match: "~/site/support/ome-artwork/(?<link>.*)$"
```

Comment:
This can probably be reduced to `~/site/support/ome-artwork/.*$` (to check with regexp) since we are not using the last part of the URL in the destination link

Reply:
Done